diagnostic.md

```diff
@@ -49,22 +48,19 @@ will take the base color and lighten it by the percentage indicated by mixing it
 with white. Write the code to tint `$cornflower` by 20%.
 
 ```scss
-// your response here
-```
+$cornflower : tint(white, 20%)}```
```

Comment:
Almost. the `tint` function already has the white passed into it. The correct way to write the `tint` function would be:
```scss
color: tint($cornflower, 20%);
```